observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,72 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
+With the following updates, you can experience improved performances of the eBPF agent, both in terms of CPU and memory: 
+
+* The eBPF agent now uses TCX webhooks instead of TC.
+* The *NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two cases: when the Network Observability eBPF agent hashmap table is full, which causes the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. 
+* Duplicated flows are now merged into a list, which is tied to a single flow data, so Loki storage demand decreases significantly. Previously, duplicated flows were stored as individual flows, and flagged as duplicate if there were multiple of the same kind of flows. You could choose to show the duplicated flows using the *Duplicate flows* Query Option. Now instead of having multiple, individual flows, there is one flow with a list of matching *Interfaces* and *Interface directions*.
+
+[NOTE]
+===
+With the duplicated flows update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any saved queries usi ng these fields need to be updated. 
+===
+
+For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
+and see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
+
+
+[id="network-observability-ebpf-collection-filtering-1.6"]
+==== eBPF collection rule-based filtering
+You can use rule-based filtering to reduce the volume of created flows. When this option is enabled, the *Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
+
+[id="network-observability-technology-preview-1.6"]
+=== Technology preview features
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
```

Comment:
```suggestion
Some features in this release are currently in Technology Preview. Technology Preview features are not intended for production use. Note the following scope of support on the Red Hat Customer Portal for these features:
```
I don't think we use "experimental" for these.

Reply:
I copied this text directly from the main OCP release notes/TP: https://docs.openshift.com/container-platform/4.15/release_notes/ocp-4-15-release-notes.html#ocp-4-15-technology-preview-tables